applications/Polkaholic.md

```diff
@@ -0,0 +1,214 @@
+
+### Team Code Repos
+
+- https://github.com/colorfulnotion/polkaholic
+- https://github.com/colorfulnotion/docs.polkaholic.io
+- https://github.com/sourabhniyogi https://www.linkedin.com/sourabhniyogi
+- https://github.com/mkchungs
+
+## Development Status :open_book:
+
+Polkaholic.io is a fully working block explorer that currently shows initial UI + API implementations, including indexing across 40+ chains and 90+ chain pairs.  We are increasing XCM Transfers coverage (95% coverage) and are developing XCM Messages + Analytics APIs.
+
+## Development Roadmap :nut_and_bolt:
+
+As part of Milestone 1 + 2, the Polkaholic team will:
+- Add _verifiability_ to all key Polkaholic.io pages: chain, block, extrinsics/txs, account with (a) browser-based JS and (b) server-side JS  
+- Add continuous _verifiability_ using server-side JS and publish % of verification checks per chain as chain-specific metadata
+- Extend polkaholic.io functionality with XCM Transfers + Message APIs, also with the same _verifiability_
+- Ensure the polkaholic.io be started with an empty Google Cloud project
+- Publish blog posts on the XCM Transfers and Messages API
+
+### Overview
+
+- **Total Estimated Duration:** 12 weeks
+- **Full-Time Equivalent (FTE):** 2.0 FTE (see [Wikipedia](https://en.wikipedia.org/wiki/Full-time_equivalent), e.g. 2 FTE)
+- **Total Costs:** 30,000 USD.
+
+| 0e. | Blog Post | We will publish blog posts that (a) explains how parachain and dapp developers can use our  XCM Transfers API to support their parachain and dapp development (b) shows how simple XCM transfers are monitored with the Polkaholic.io UI |
+
+### Milestone 1 — Implement Verifiability for Chain, Block, Extrinsics, Accounts
+
+- **Estimated duration:** 4 weeks
+- **FTE:**  2.0 FTE
+- **Costs:** 10,000 USD
+
+We will deliver verifiable data concerning: chain state, blocks, extrinsics, and accounts:
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0A. | License | GNUv3 |
+| 0B. | Documentation | We will provide: (a) **inline documentation** of the core crawling and indexing processes (b) how to setup polkaholic.io from an empty Google Cloud project |
+| 1. | UI Augmentation: Browser-based JS for Verifiability | For all key pages, we will add browser-based code to verify results by utilizing RPC calls. |  
+| 2. | UI Augmentation: Server-side JS for Verifiability   | For all key pages, we will publish similar server-side code that also verifies results. |  
```

Comment:
Can you be more specific about what you mean by "key pages" and which RPC calls are available?  Same with the other deliverables - just looking to be more exact so that we can test it appropriately upon milestone delivery.

Other than that, I'd be very happy to approve this. The interface you already have is very useful.

Reply:
@laboon To get more specific, we mapped the polkaholic.io site / API into approximately 20 "verification functions" here in [this sheet](https://docs.google.com/spreadsheets/d/1w9xIhyHgB05PgXEVDMyazRYkYvBD3ETaIbWb8X2DKYc/edit#gid=323796891).  All the verification functions will just do the polkadot JS API calls RPC in a way that everyone is familiar with -- eg for verifying block data following a path like  [this](https://github.com/colorfulnotion/polkaholic/blob/main/substrate/test/testGetAuthor.js#L18-L26) and then having "true" or "false" come back if the supplied inputs to the verification function match the data returned by the set of RPC call.

Reply:
Thank you, all makes sense to me.